src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/ExpandedMenuToggle.tsx

```diff
@@ -44,52 +48,61 @@ const text = ({ showExpandedMenu }: { showExpandedMenu: boolean }) => css`
         display: inline-block;
         height: 8px;
         margin-left: 6px;
-        transform: ${showExpandedMenu
-            ? 'translateY(1px) rotate(-135deg)'
-            : 'translateY(-3px) rotate(45deg)'};
+
+        /*
+            IMPORTANT NOTE:
+            we need to specify the adjacent path to the a (current) tag
+            to apply styles to the nested tabs due to the face we use ~
+            to support NoJS
+        */
+        transform: translateY(-3px) rotate(45deg);
+        ${`#${CHECKBOX_ID}:checked ~ & {
+            transform: translateY(1px) rotate(-135deg);
+        }`}
+
         transition: transform 250ms ease-out;
         vertical-align: middle;
         width: 8px;
     }
     :hover:after {
-        transform: ${showExpandedMenu
-            ? 'translateY(-2px) rotate(-135deg)'
-            : 'translateY(0) rotate(45deg)'};
+        /*
+            IMPORTANT NOTE:
+            we need to specify the adjacent path to the a (current) tag
+            to apply styles to the nested tabs due to the face we use ~
+            to support NoJS
+        */
+        transform: translateY(0) rotate(45deg);
+        ${`#${CHECKBOX_ID}:checked ~ & {
+            transform: translateY(-2px) rotate(-135deg);
+        }`}
     }
 `;
 
 interface Props {
     display: Display;
-    showExpandedMenu: boolean;
-    toggleExpandedMenu: (value: boolean) => void;
     ariaControls: string;
+    CHECKBOX_ID: string;
 }
 
 export const ExpandedMenuToggle = ({
     display,
-    toggleExpandedMenu,
     ariaControls,
-    showExpandedMenu,
+    CHECKBOX_ID,
 }: Props) => (
     <>
         <VeggieBurger
             display={display}
-            showExpandedMenu={showExpandedMenu}
-            toggleExpandedMenu={toggleExpandedMenu}
+            CHECKBOX_ID={CHECKBOX_ID}
             ariaControls={ariaControls}
             key="VeggieBurger"
         />
-        <button
+        <LabelButton
             className={openExpandedMenu(display)}
-            onClick={() => toggleExpandedMenu(!showExpandedMenu)}
-            aria-controls={ariaControls}
-            key="OpenExpandedMenuButton"
-            data-link-name={`nav2 : veggie-burger : ${
```

Comment:
We need to preserve this logic

Reply:
It is unclear to me what this does, and how to preserve this?

Reply:
They're data attributes used for tracking

Reply:
data-link-name and data-component attributes are automatically hooked into by the tracker.js script from Ophan, as Oliver mentions they're required and they need to be the exact same as Frontend.

Reply:
I am going at add a prop to `NoJSButton` called `dataLinkName` because this component is NoJS `showExpandedMenu ? 'show' : 'hide'` wont be possible

Reply:
also, this is not a veggie burger? this is just a Nav. Should I remove `veggie-burger` from `data-link-name`? and keep it for the label in the `<VeggieBurger />` ?

Reply:
I don't think so as to match up with Frontend it needs to be the same linkname when clicked and it's the same element for both desktop and mobile - 
![image](https://user-images.githubusercontent.com/638051/80374487-9f71f280-888e-11ea-8cc4-2a853257ee32.png)

Reply:
https://github.com/guardian/dotcom-rendering/pull/1399/files#diff-6b05eaaad73e58efb4df4b1cbe70e9d3R109

` data-link-name="nav2 : veggie-burger"` has been added for both buttons